content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewers:
+- msau42
+- xing-yang
+title: Volume Attributes Classes
+content_type: concept
+weight: 40
+---
+<!-- overview -->
+
+This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
+[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+in Kubernetes.
+
+<!-- body -->
+
+## Introduction
```

Comment:
(nit) the introduction was:

> This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
> [volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
> in Kubernetes.

```suggestion
```

Reply:
What do you mean here? Remove the introduction title completely or move it to above?

Reply:
And do we change the StorageClass page as well if we change the format here?

Reply:
I recommend removing the “Introduction” heading; we avoid having a heading with that name.

No need for _this_ PR to improve the StorageClass docs.